ui.content.sample/src/main/content/jcr_root/content/experience-fragments/wknd/language-masters/en/site/sign-in/sign-in-page/.content.xml

```diff
@@ -0,0 +1,89 @@
+                    jcr:primaryType="nt:unstructured"
+                    jcr:title="Welcome Back"
+                    sling:resourceType="wknd/components/title"
+                    type="h3"/>
+                <sign_in_form
+                    jcr:primaryType="nt:unstructured"
+                    sling:resourceType="wknd/components/form/sign-in-form"
+                    actionUrl="/system/sling/login/j_security_check"
+                    redirectUrl="/content/wknd/us/en.html">
+                    <container
+                        jcr:primaryType="nt:unstructured"
+                        sling:resourceType="wknd/components/container">
+                        <field-1
+                            jcr:lastModified="{Date}2019-10-21T20:09:51.128-07:00"
+                            jcr:lastModifiedBy="admin"
+                            jcr:primaryType="nt:unstructured"
+                            jcr:title="USERNAME"
+                            sling:resourceType="wknd/components/form/text"
+                            helpMessage="USERNAME"
+                            hideTitle="true"
+                            name="j_username"
+                            rows="1"
+                            type="text"
+                            usePlaceholder="true"/>
+                        <field-2
+                            jcr:primaryType="nt:unstructured"
+                            jcr:title="PASSWORD"
+                            sling:resourceType="wknd/components/form/text"
+                            helpMessage="PASSWORD"
+                            hideTitle="true"
+                            name="j_password"
+                            rows="1"
+                            type="password"
+                            usePlaceholder="true"/>
+                        <text-1
+                            jcr:primaryType="nt:unstructured"
+                            sling:resourceType="wknd/components/text"
+                            text="&lt;p>&lt;a href=&quot;#&quot;>FORGOT YOUR PASSWORD?&lt;/a>&lt;/p>&#xa;"
+                            textIsRich="true"/>
+                        <button-1
+                            jcr:primaryType="nt:unstructured"
+                            jcr:title="SIGN IN"
+                            sling:resourceType="wknd/components/form/button"
+                            type="submit"/>
+                        <hidden
+                            jcr:created="{Date}2019-10-22T15:43:57.901-07:00"
+                            jcr:createdBy="admin"
+                            jcr:lastModified="{Date}2019-10-22T16:16:49.541-07:00"
+                            jcr:lastModifiedBy="admin"
+                            jcr:primaryType="nt:unstructured"
+                            sling:resourceType="wknd/components/form/hidden"
+                            id="sling-auth-redirect-signin"
+                            name="sling.auth.redirect"
+                            value="/"/>
```

Comment:
havent tried it, but could you put `.` or `./` here instead of relying on JS to inject the current path value?

Reply:
@davidjgonzalez `.` or `./` does not work, unfortunately. I don't love injecting it with javascript but the modal gets loaded from an experience fragment (so it doesn't have the context of the current page)